docs/CommandLineOptions.html

```diff
@@ -371,6 +371,10 @@ <h2>Options allowed on the command line and in the config file</h2>
 Can help performance, but only use it if you know you will not 
 need to access the input files again soon.
 
+<tr id='Reads.palindromicReads.skipFlagging'>
+<td><code>--Reads.palindromicReads.skipFlaging</code><td class=centered><code>False</code><td>
```

Comment:
The option name is mispelled (missing a `g`).

Reply:
Good catch. Will fix.